tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -55,6 +68,211 @@ oe_result_t enclave_identity_verifier(oe_identity_t* identity, void* arg)
     return result;
 }
 
+#ifdef OE_USE_OPENSSL
+static oe_result_t generate_rsa_pair(
+    uint8_t** public_key,
+    size_t* public_key_size,
+    uint8_t** private_key,
+    size_t* private_key_size)
+{
+    oe_result_t result = OE_FAILURE;
+    uint8_t* local_public_key = NULL;
```

Comment:
style nit: use the C++ keyword nullptr instead of NULL in .cpp files

Reply:
Changed to .cpp to .c (this file don't need to be .cpp).

Reply:
I revert the change (from .cpp to .c) because so will make the entire file as a newly added file and reviewers cannot know what part of the file are introduced by this PR. We can make a follow-up PR to change the file extension.